api/accounts.js

```diff
@@ -1,20 +1,21 @@
 const accounts = require('../lib/api/accounts');
```

Comment:
it's better to use `strict` flag on top of your codes if you want to use ES2015.
read this article: https://nodejs.org/en/docs/es6/

Reply:
Since we apply the same restrictions though Eslint rules I don't see it vital.